### PR TITLE
docs(tools): update broken link to dashboard

### PR DESCRIPTION
**Pull request by MarceloDamian**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

While looking through some documents, I found a link that redirects to a faulty site. Thank you so much.

### CODE REVIEW

**Comment by raisedadead** [comment]
The tools dashboard has moved to: https://tools.freecodecamp.org/dashboard/, but it's not completely maintained at the minute. Other than being used by some of our mods. 

We are working towards overhauling it. 

I think for now, we should remove the link and add a notice along the lines of:


> We are working on creating a tools dashboard for for helping you review PRs and translations. Hangout with us in the [contributors chat room](https://gitter.im/FreeCodeCamp/Contributors) to learn more.

**Comment by MarceloDamian** [comment]
Sounds good! Should I commit the new changes to this pull request?

**Comment by raisedadead** [comment]
> Should I commit the new changes to this pull request?

Yes please.

**Comment by MarceloDamian** [comment]
Committed the new changes and the link to the contributor's chat room! Thank you so much!!!

File: tools/dashboard/README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 ![freeCodeCamp Social Banner](https://s3.amazonaws.com/freecodecamp/wide-social-banner.png)
 
-# contribute
+# Contribute
+
+Tools to help maintain [freeCodeCamp.org](https://www.freecodecamp.org)'s Open Source Codebase on GitHub. We are currently working on creating a tools dashboard for helping you review PRs and translations. Hangout with us in the [contributor's chat room](https://gitter.im/FreeCodeCamp/Contributors) to learn more. 
+
+
+
+
```
Comment (raisedadead) [inline tools/dashboard/README.md:9]:
```suggestion
```

Can you cleanup these extraneous lines. Thanks.

**Review by raisedadead — CHANGES_REQUESTED** [review]

File: tools/dashboard/README.md (hunk 1 above)
Comment (MarceloDamian) [reply tools/dashboard/README.md]: Committed the new changes! Thank you.

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
Thanks 👍

**Pull request merged**